articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+
+### Styling in CSS
+A simple google search will show you that there three different styling methods in CSS each different from the other. Each level has its application and use scenario depending on what gets the job done.
+
+  1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+    </body>
+  </html>
+  ~~~
+
+  2. **Page-Level styles:** Page-level styles are defined at the header area of the HTML file. Unlike local styles, you can customize all the similar tags at once thereby saving time and maintaining uniformity.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+      <meta charset="utf-8">
+      <style type="text/css">
+         body{
+           color: yellow;
+           background-color: red;
+         }
+         p{
+           color: red;
+           background-color: yellow;
+         }
+      </style>
+    </head>
+    <body>
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+  ~~~
+
+  3. **External Styles** In this case, the styles used for the webpage are located in a completely different file. They are linked to the HTML file depending on which CSS file (Style) best fits the webpage. When developing a website with multiple pages this styling usually comes in handy.
+
+  ~~~html
+  <!DOCTYPE html>
+  <html>
+    <head>
+        <meta charset="utf-8">
+        <title>External Styles</title>
+        <link rel="stylesheet" type="text/css" href="css/myStyle.css"/>
+    </head>
+    <body>
+      <h1>Heading</h1>
+        <p>This is an example of External CSS</p>
+    </body>
+  </html>
+  ~~~
+  ~~~CSS
+  /*This is the CSS file called myStyle.css*/
+  body{
+    background-color: black;
+    color: white;
+  }
+  p{
+    color: purple;
+  }
+  ~~~
+
+### When to use each Style
+With these three options for styling in CSS, it becomes rather difficult to identify the most appropriate when building a webpage.
```

Comment:
Not true. There are well-known best practices that say avoid internal CSS as a rule of thumb and favor external.

Reply:
Best practices should actually be mentioned and highlighted.